release/pkg/utils/utils.go

```diff
@@ -83,3 +77,31 @@ func GetManifestFilepaths(devRelease bool, bundleNumber int, kind, branch string
 	}
 	return manifestFilepath
 }
+
+func SortArtifactsMap(m map[string][]releasetypes.Artifact) []string {
```

Comment:
I don't see any relationship between the new and existing functions. As with many `common`, `utils` and `base` packages that represent arbitrary logic, this looks like its set to become Sakaar. 

Suggest we don't add to this and instead create a nested packaged with a concrete purpose. E.g. `/utils/git` and `/utils/uri`.

Reply:
I had these as miscellaneous functions which served as nothing more than utilities to other functions. I can do the nested approach if that would be more idiomatic.

Reply:
As a rule of thumb we should avoid `util` packages that have undefined purpose. 

https://dave.cheney.net/practical-go/presentations/qcon-china.html#_avoid_package_names_like_base_common_or_util
https://github.com/go-kit/kit/tree/master/util/conn

Reply:
Split into nested packages, thanks for the references!